docs/CONTRIBUTING.md

```diff
@@ -16,16 +16,18 @@
 
 1. Install Node.js and `npm` on your development machine by following the instructions on the Node.js website.
```

Comment:
We need to mention pre-requisitives beforehand, like we need Node `v18.14.0` and `npm`. To use a local backend server the user might even be required to have `MongoDB` installed. So a section, by the name **pre-requisitives** could mention it at the start of this guide.